app/src/scenes/SubCategories/index.tsx

```diff
@@ -88,40 +101,62 @@ class SubCategories extends React.Component<
     }
     return (
       <Router>
-        <Title level={2} className={styles.mainContent}>
-          {title}
-        </Title>
-        <Menu
-          mode="horizontal"
-          onClick={this.handleClick}
-          selectedKeys={[this.state.current]}
-        >
-          {this.state.subCategories.map((subCategory) => {
-            // Replace spaces and slashes from the subcategory name to include it on the URL
-            const categoryName = title
-              .trim()
-              .replace(/\s+|\//g, '-')
-              .toLowerCase();
-            return (
-              <Menu.Item key={subCategory.id} className={styles.menuItem}>
-                <Link
-                  to={`/academix/${this.CategoryId}/${categoryName}/${subCategory.id}`}
-                >
-                  {subCategory.translations[0].name}
-                </Link>
-              </Menu.Item>
-            );
-          })}
-        </Menu>
-        <Switch>
-          <Route
-            path="/academix/:categoryId/:categoryName/:subCategoryId"
-            component={Items}
-          />
-        </Switch>
+        <Row className={styles.mainContent}>
+          <Col md={20}>
```

Comment:
md 20? cant we use 24?

Reply:
It should be 20, check mainContent CSS. Instead of creating two 3 columns to keep the alignment like in home page, I added a justify-content center to the main content class